uiowa/uiowa
# Feature: Lists and heading bug fixes

In this pull request:

- Resolves https://github.com/uiowa/uiowa/issues/3637
- Resolves https://github.com/uiowa/uiowa/issues/3636
- Resolves https://github.com/uiowa/uiowa/issues/3635
- Resolves https://github.com/uiowa/uiowa/issues/3459
- Updates headline text style description from `'Bold serif, highlighted'` to `'Bold serif, underlined'` for a more accurate description. 
- Adds a `<p>` tag on views article block no result text for consistency across all blocks.
- Removed custom headline sizing from events block title as @mark-bennett-uiowa notes in https://github.com/uiowa/uiowa/issues/2871. 
- Resolves https://github.com/uiowa/uiowa/issues/3640
- Resolves https://github.com/uiowa/uiowa/issues/3643
- Resolves https://github.com/uiowa/uiowa/issues/2871
- Resolves https://github.com/uiowa/uiowa/issues/2433

# How to test
```
blt ds --site=sandbox.uiowa.edu
yarn workspace uids_base gulp --development
drush @sandbox.local uli
drush @sandbox.local cr
```
- Go to https://sandbox.local.drupal.uiowa.edu/list-and-heading-bugs and compare against https://sandbox.prod.drupal.uiowa.edu/list-and-heading-bugs
   - Events and Aggregator no results blocks are applying card styles to the block wrapper https://github.com/uiowa/uiowa/issues/3637
   - Events heading is not displaying if there are no results listed in the events block https://github.com/uiowa/uiowa/issues/3636
   - Grids with a more-link do not have a margin to create separation from grid item https://github.com/uiowa/uiowa/issues/3635
   - Component headings not consistent  https://github.com/uiowa/uiowa/issues/3459
   - People list block has extra top margin due to views-element-container spacing https://github.com/uiowa/uiowa/issues/3640
   - People list block headline missing `block__headline` class: https://github.com/uiowa/uiowa/issues/3643
- Edit a block headline and verify that the description has changed from from `'Bold serif, highlighted'` to `'Bold serif, underlined'` .
- Go to https://sandbox.local.drupal.uiowa.edu/list-and-heading-bugs and compare the people and articles no results text against https://sandbox.prod.drupal.uiowa.edu/list-and-heading-bugs.  The local site should have paragraph tags on the no results text.
